moveit_ros/moveit_servo/package.xml

```diff
@@ -23,6 +23,7 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <depend>moveit_common</depend>
 
+  <depend>generate_parameter_library</depend>
```

Comment:
Add this in alphabetical order, so before `geometry_msgs`

Reply:
Does the `moveit_common` also need to be moved ?
Or is that put there on purpose ?

Reply:
Not sure. It seems to be at the top for most moveit packages, so maybe just leave that one there?

Reply:
Did not see any issues after I moved it.
Is there anyone who can clarify on this?
Feels weird to leave it like that now.

Reply:
I'd be surprised if order mattered.

Personally, I'd prefer alphabetized but no strong opinion

Reply:
I don't think there's any issues with rearranging things in `package.xml`. Typically they are alphabetized, but I'm sure there was a decision to have `moveit_common` at the top for some visibility reason. Don't worry about it.